doc_src/development_environment.md

```diff
@@ -80,3 +80,19 @@
             - Milliseconds to wait until gear action finishes. Defaults to `10000`. See also [controller](https://hexdocs.pm/antikythera/controller.html).
         - `GEAR_PROCESS_MAX_HEAP_SIZE`:
             - Maximum size of per-process heap memory in words. Defaults to `50000000` (400MB in 64-bit architecture).
+
+- If you develop mix task, you can use the following environment variables:
+    - Runtime environment variables:
```

Comment:
Maybe, This block is not necessary because `ANTIKYTHERA_MIX_TASK_MODE` is refered at compile time.

Reply:
Good catch!
I think `running_with_release?/0` and `running_in_cloud?/0` should use `ANTIKYTHERA_MIX_TASK_MODE` at runtime because they use `ANTIKYTHERA_RUNTIME_ENV` at runtime.
Fixed at https://github.com/access-company/antikythera/pull/125/commits/9973790f87b7c07f2fd9f51cc717091d8c555e44